src/Microsoft.AspNetCore.Mvc.Testing/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -17,7 +17,9 @@
 
     <Target Name="_ResolveMvcTestProjectReferences" DependsOnTargets="ResolveReferences">
       <ItemGroup>
-        <_ContentRootProjectReferences Include="@(ReferencePath)" Condition="'%(ReferencePath.ReferenceSourceTarget)' == 'ProjectReference'" />
+        <_ContentRootProjectReferences
+          Include="@(ReferencePath)"
+          Condition="'%(ReferencePath.ReferenceSourceTarget)' == 'ProjectReference' and '%(ReferencePath.TargetFrameworkIdentifier)' != '.NETStandard'" />
```

Comment:
Is checking for '.NETStandard` the best thing to check for here?  I would have assumed we would check for `HasRuntimeOutput`.  (But I am unsure how to get that property from a Reference.)

Reply:
@eerhardt Its still an approximation but we haven't found a case where it doesn't work. We couldn't find anything better to key this off when we were looking at it. Maybe this information (HasRuntimeOutput) is something we can surface in the next release?

Reply:
The case I was thinking of was referencing a `netcoreapp2.0` (or `1.0` or `1.1`) library from a `netcoreapp2.1` app.

In this case, the assemblies you compiled the `netcoreapp2.0` library against are going to be different than the assemblies that are published for `netcoreapp2.1`.  You may run into similar problems that assemblies can't be found at runtime.  (note the "**may**", I'm not 100% sure this is the case).